### PR TITLE
Lza/board

### PR DESCRIPTION
**Pull request by zdlm**
There are no buttons and tabs in this version of boar. 
I will give collaboration team to use this one first. 
Make buttons and tabs later
@esmailpour-hosein @aleks-elkin

### CODE REVIEW

File: packages/components/aside/src/aside.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	min-height: 100vh;
 	width: var(--ts-aside);
 	background-color: var(--ts-color-white);
-	box-shadow: 0 0 3px var(--ts-shadow-color);
+	box-shadow: 0 0 3px var(--ts-color-slate-dark);
```
Comment (esmailpour-hosein) [inline packages/components/aside/src/aside.css]: Shouldn't we change the `ts-shadow-color` to slate-dark?

File: packages/components/board/src/board.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		static get properties() {
+			return {
+				dir: { type: String, reflect: true },
+				title: { type: String, attribute: 'data-title' }
+			};
+		}
+
+		constructor() {
+			super();
+			this.title = '';
```
Comment (esmailpour-hosein) [inline packages/components/board/src/board.js]: Why do we need this?

File: packages/components/board/src/board.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+				</header>
+			`;
+		}
+
+		render() {
+			return html`
+				<div dir="${this.direction}" class="board-container">
+					${this.header}
+					<main class="board-main">
+						<slot name="main"></slot>
```
Comment (esmailpour-hosein) [inline packages/components/board/src/board.js]: What if we don't put a name for it?

File: packages/components/board/stories/board.happo.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	const properties = {
+		dir: { rtl: 'rtl' },
+		title: { Title: 'Title' }
+	};
+
+	const options = {
+		columns: 5,
+		persistent_props: {
+			errorMessage: 'Failed message'
+		}
```
Comment (esmailpour-hosein) [inline packages/components/board/stories/board.happo.js]: This need to be removed

File: packages/components/board/stories/board.stories.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+import '@tradeshift/elements.board';
+
+storiesOf('ts-board', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const title = text('My Board', 'Title');
+
+		return html`
+			<ts-board data-title="${title}">
```
Comment (esmailpour-hosein) [inline packages/components/board/stories/board.stories.js]: Please also add `dir` prop 👍

File: packages/components/board/stories/board.stories.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+				This my first board.
```
Comment (esmailpour-hosein) [inline packages/components/board/stories/board.stories.js]: Can you please make this part also dynamic so people can provide `html` in the slot?

File: packages/core/src/vars.css [hunk 7]
```diff
@@ -155,7 +155,7 @@
 	--ts-progress-bar-height: 8px;
 	--ts-box-width-small: 280px;
 	--ts-box-width-medium: 360px;
-	--ts-shadow-color: var(--ts-color-slate-dark);
+	--ts-shadow-color: var(--ts-color-gray-lightest);
```
Comment (esmailpour-hosein) [inline packages/core/src/vars.css]: Now I saw this one 🤔 Was the `aside` the only place we used this shadow-color?

File: packages/components/aside/src/aside.css (hunk 1 above)
Comment (zdlm) [reply packages/components/aside/src/aside.css]: The shadow in the aside is different with the current shadow in the board.
The slate-dark is only for aside, And the light-gray is for the common shadow.

File: packages/components/board/src/board.js (hunk 2 above)
Comment (zdlm) [reply packages/components/board/src/board.js]: 👍

File: packages/components/board/src/board.js (hunk 3 above)
Comment (zdlm) [reply packages/components/board/src/board.js]: I will add more slots. such as `tab`, `Button`.
Let's keep it right now.

File: packages/components/board/stories/board.happo.js (hunk 4 above)
Comment (zdlm) [reply packages/components/board/stories/board.happo.js]: 👍

File: packages/components/board/stories/board.stories.js (hunk 5 above)
Comment (zdlm) [reply packages/components/board/stories/board.stories.js]: 👍

File: packages/core/src/vars.css (hunk 7 above)
Comment (zdlm) [reply packages/core/src/vars.css]: Yes. that is true

**Comment by esmailpour-hosein** [comment]
Please check happo for IE11, It doesn't seems right.

**Pull request merged**